examples/rendercontrol_opengl/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Gui OpenGL Qml Quick QuickControls2)
+
+set(SOURCES
+    ../images.qrc
+    rendercontrol.qrc
+    main.cpp
+    window_singlethreaded.cpp
+)
+
+qt_add_executable(rendercontrol_openglexample WIN32 ${SOURCES})
```

Comment:
There's no `qt_add_executable` in Qt5, please use the normal `add_executable` instead.

Reply:
Fixed in b9f8d1dcc4286756c0220b3961733d4f994890a1